### PR TITLE
Completed Task 3

### PR DESCRIPTION
**Pull request by kesari007**
`Solves #6`

### CODE REVIEW

**Review by Jigar3 — CHANGES_REQUESTED** [review]
Upload the data you got from scraping as well

**Comment by kesari007** [comment]
@Jigar3 upload as in? The BeautifulSoup object `soup` already has all the data in it though.

**Comment by Jigar3** [comment]
Save the object in a txt or JSON file and upload that

**Comment by kesari007** [comment]
okay sure.

**Comment by kesari007** [comment]
@Jigar3 I have made the changes, Please review it now.

File: Task 3/data.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
+{
+    "Title": "Roma, The Favourite lead Oscars 2019 nominations with 10 each",
+    "Content": "Oscar-winning director Alfonso Cuar\u00f3n's Roma and Yorgos Lanthimos' The Favourite have received 10 Oscars 2019 nominations each, including those for Best Picture and Production Design. Further, Bradley Cooper's A Star Is Born and Adam McKay's Vice have been nominated for eight Oscars each. Black Panther, which received seven nominations, became the first superhero movie to be nominated for Best Picture. ",
+    "Image_Url": "https://assets.inshorts.com/inshorts/images/v1/variants/jpg/m/2019/01_jan/22_tue/img_1548166249334_475.jpg?resize=*:400px"
+}{
```
Comment (Jigar3) [inline Task 3/data.json]: Add comma after each entry

**Review by Jigar3 — CHANGES_REQUESTED** [review]

File: Task 3/data.json (hunk 1 above)
Comment (kesari007) [reply Task 3/data.json]: Done @Jigar3

**Comment by kesari007** [comment]
@Jigar3 Please review now.

**Pull request closed without merging**